tests/tx/test_merged_mining.py

```diff
@@ -1410,3 +1412,7 @@ async def submit_block(self, block: bytes) -> str:
         stub = 'high-hash'
         await asyncio.sleep(self.response_delay)
         return stub
+
+    async def validate_address(self, address: str) -> Dict:
```

Comment:
Seems to be from the other PR.

Reply:
I've proposed a similar API, but this is the one from Bitcoin that inspired ours. I'm now using it (on both Bitcoin and DigiByte), to simplify address validation, support more address types, and not have to build the output script manually.

Reply:
Excellent